### PR TITLE
Automate steps and update document 

### LINKED ISSUE #306: Automate to generate arm deploy script and update document

### PR DESCRIPTION
**Pull request by NT-D**
# Type of PR

- [x] Documentation changes
- [x] Code changes
- [ ] Test changes
- [ ] CI-CD changes
- [ ] GitHub Template changes

## Purpose

- Based on the feedback, update document
- Automate to generate Azure Data Factory ARM template deployment script

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Author pre-publish checklist:

- [ ] Added test to prove my fix is effective or new feature works
- [ ] No PII in logs
- [ ] Made corresponding changes to the documentation

## Issues Closed or Referenced

- Closes #306 

- References #issue_number

### CODE REVIEW

File: e2e_samples/dataset_versioning/datafactory/README.md [hunk 1]
```diff
@@ -2,14 +2,22 @@
...
+
 ## How to deploy it
+Run command to deploy business logic into Azure Data Factory ([document](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/deploy-cli#deploy-local-template-or-bicep-file)). You can utilize terraform script `arm_deploy_script` output which you get in provisioning step.
 
-1. Provision Azure Resources by IaC (Terraform)
-1. [Install the Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli)
-1. Run command to deploy ([document](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/deploy-cli#deploy-local-template-or-bicep-file))
+### Command
+```bash
+az deployment group create --name {Your deployment name. Anyname is OK} --resource-group {Your resource group name} --template-file ./arm_template/arm_template.json --parameters factoryName="{Your data factory name}" KeyVault_properties_typeProperties_baseUrl="{Your key vault url}" AzureBlobFS_properties_typeProperties_serviceEndpoint="{Your blob storage url}"
```
Comment (jmostella) [inline e2e_samples/dataset_versioning/datafactory/README.md:15]: you can remove `Anyname is OK`

File: e2e_samples/dataset_versioning/infra/README.md [hunk 2]
```diff
@@ -70,6 +70,15 @@ terraform plan -var="rg_name=your_rg_name" -var="app_name=your_app_name" -out=/t
 terraform apply /tmp/myapp
 ```
 
+After terraform apply completes successfully, you can see output similar to following example. Please copy `arm_deploy_script` value for later process. You'll use it for deploying Azure Data Factory Logic.
+```
```
Comment (jmostella) [inline e2e_samples/dataset_versioning/infra/README.md:74]: Need to add a language. I use `console` for output

**Review by jmostella — CHANGES_REQUESTED** [review]

**Review by jmostella — APPROVED** [review]
LGTM thanks!

File: e2e_samples/dataset_versioning/datafactory/README.md [hunk 3]
```diff
@@ -2,14 +2,22 @@
 
 This folder is for Data pipeline which copy data from SQL DB to Azure Storage (Delta Lake). We are using Azure Data Factory to implement pipeline.
 
+## prerequisites
+Expect you've already finishes previous set-up including..
+- Provision Azure Resources by IaC (Terraform)
+- [Install the Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli)
```
Comment (icchan) [inline e2e_samples/dataset_versioning/datafactory/README.md:7]: Maybe just a small thing.
But actually, we need to install the Azure cli before we can provision the resources with Terraform.

**Pull request merged**